js/packages/phoenix-client/package.json

```diff
@@ -81,13 +81,7 @@
   "dependencies": {
     "@arizeai/openinference-semantic-conventions": "^1.1.0",
     "@arizeai/openinference-vercel": "^2.3.1",
-    "@opentelemetry/api": "^1.9.0",
```

Comment:
We can probably move this to dev deps to keep the types around, or re-export from phoenix otel

Reply:
Yeah, re-export makes the most sense to me given it's honest and we want phoenix-otel to be a self-contained module